bpffs/fs.go

```diff
@@ -8,7 +8,7 @@ import (
 const (
 	BPFFSPath = "/sys/fs/bpf"
 	// https://github.com/coreutils/coreutils/blob/v8.27/src/stat.c#L275
-	FsMagicBPFFS = 0xCAFE4A11
+	FsMagicBPFFS = int64(0xCAFE4A11)
```

Comment:
Btw, the canonical definition of the constant is in:
https://github.com/torvalds/linux/blob/v4.8/include/uapi/linux/magic.h#L80
```
#define BPF_FS_MAGIC		0xcafe4a11
```